### PR TITLE
added anime class and initial tests

### PR DESCRIPTION
**Pull request by cgenglish**

### CODE REVIEW

File: AnimeTest.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import org.junit.Test;
+
+public class AnimeTest {
+    @Test
+    public void testGetMainChar() {
```
Comment (Merlissa09) [inline AnimeTest.java:8]: Noticed that the test names are not following our typical naming convention. Would we be able to update to follow per this standard? https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices

**Review by Merlissa09 — APPROVED** [review]

File: AnimeTest.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        //Arrange
+        Anime naruto = new Anime();
+
+        //Action
+        String test = naruto.getMainChar();
```
Comment (cmcknight14) [inline AnimeTest.java:13]: maybe change to Sasuke for more dynamic character arc?

File: AnimeTest.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+
+    @Test
+    public void testSetVillain() {
+         //Arrange
+         Anime naruto = new Anime();
+
+         //Action
+         String test = naruto.getVillain();
+         naruto.setVillian("Madara");
```
Comment (cmcknight14) [inline AnimeTest.java:91]: allmighty push to Pain

**Review by cmcknight14 — COMMENTED** [review]
manga was better

**Review by cmcknight14 — APPROVED** [review]

**Pull request merged**